lib/Data.js

```diff
@@ -34,21 +36,17 @@ export class Suite {
     return this.tests.concat(childSuiteTests)
   }
 
-  get runtime () {
-    var status = this.status
-
-    if (status === 'skipped' || status === undefined) {
+  getRuntime () {
```

Comment:
Is there any reason to export this method? If the object isn't expected to change after it was created, we might as well keep this private (with some workaround, since ES6 classes don't have private scope for methods).

Same for `getStatus()`